### PR TITLE
Chinese doc for auto analyze auto tune support

### PR DESCRIPTION
**Pull request by BornChanger**
Signed-off-by: BornChanger <dawn_catcher@126.com>

### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- Added the description about Quota Limiter for background requests

- Added the quota-related configurations:` background-cpu-time`, `background-write-bandwidth`, `background-read-bandwidth` and `enable-auto-tune`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/tikv/tikv/pull/12679
- Other reference link(s):

### Do your changes match any of the following descriptions?
N/A

### CODE REVIEW

**Review by tabokie — APPROVED** [review]

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1685,9 +1685,37 @@ Raft Engine 相关的配置项。
...
+
++ 限制后端事务写入的带宽，这是一个软限制。
++ 默认值：0KB（即无限制）
+
+### `background-read-bandwidth`（从 v6.2.0 版本开始引入）
+
++ 限制后端事务读取数据和 Coprocessor 读取数据的带宽，这是一个软限制。
++ 默认值：0KB（即无限制）
+
+### `support-auto-tune`（从 v6.2.0 版本开始引入）
```
[the remark this answers is not in the document]
Comment (BornChanger) [reply tikv-configuration-file.md]: > enable-auto-tune.

fixed.

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1685,9 +1685,37 @@ Raft Engine 相关的配置项。
...
 + 默认值：0KB（即无限制）
 
+当 TiKV 部署的机型资源有限（如 4v CPU，16 G 内存）时，如果 TiKV 后台处理的计算或者读写请求量过大，以至于占用 TiKV 前台处理请求所需的 CPU 资源，最终影响 TiKV 性能的稳定性。此时，你可以使用后台限流相关的 quota 配置项以限制后台各类请求占用的 CPU 资源。触发该限制的请求会被强制等待一段时间以让出 CPU 资源。具体等待时间与新增请求量相关，最多不超过 [`max-delay-duration`](#max-delay-duration从-v600-版本开始引入) 的值。
+
+> **警告：**
+>
+> - 后台限流是 TiDB 在 v6.2.0 中引入的实验特性，不建议在生产环境中使用。
+> - 该功能仅适合在资源有限的环境中使用，以保证 TiKV 在该环境下可以长期稳定地运行。如果在资源丰富的机型环境中开启该功能，可能会导致读写请求量达到峰值时 TiKV 的性能下降的问题。
+
+### `background-cpu-time`（从 v6.2.0 版本开始引入）
```
Comment (TomShawn) [inline tikv-configuration-file.md:1695]: 建议把 quota 下的配置项版本标识都改为 `<span>`，之前是因为前端网站出了问题，暂时不支持 `<span>` 了，所以临时用了（从 v6.0.0 版本开始引入）
```suggestion
### `background-cpu-time` <span class="version-mark">从 v6.2.0 版本开始引入</span>
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1685,9 +1685,37 @@ Raft Engine 相关的配置项。
...
+
+### `background-read-bandwidth`（从 v6.2.0 版本开始引入）
+
++ 限制后端事务读取数据和 Coprocessor 读取数据的带宽，这是一个软限制。
++ 默认值：0KB（即无限制）
+
+### `enable-auto-tune`（从 v6.2.0 版本开始引入）
+
++ 是否支持 quota 动态调整；如果打开，后端请求的限制 quota 会根据 TiKV 实例的负载情况动态调整。
++ 默认值： false （即不支持动态调整）
```
Comment (TomShawn) [inline tikv-configuration-file.md:1714]:
```suggestion
+ 是否支持 quota 动态调整。如果打开该配置项，TiKV 会根据 TiKV 实例的负载情况动态调整对后端请求的限制 quota。
+ 默认值：false （即关闭动态调整）
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1685,9 +1685,37 @@ Raft Engine 相关的配置项。
...
+
 ### `max-delay-duration`（从 v6.0.0 版本开始引入）
```
Comment (TomShawn) [inline tikv-configuration-file.md:1716]: 能否放到 `foreground-cpu-time` 前面呢？

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1660,7 +1660,7 @@ Raft Engine 相关的配置项。
 
 ## quota
 
-用于前台限流 (Quota Limiter) 相关的配置项。
+用于前后台限流 (Quota Limiter) 相关的配置项。
```
[the remark this answers is not in the document]
Comment (BornChanger) [reply tikv-configuration-file.md:1673]: Good suggestion

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1685,9 +1685,37 @@ Raft Engine 相关的配置项。
 + 限制事务读取数据和 Coprocessor 读取数据的带宽，这是一个软限制。
 + 默认值：0KB（即无限制）
```
[the remark this answers is not in the document]
Comment (BornChanger) [reply tikv-configuration-file.md:1696]: > 

Good suggestion

File: tikv-configuration-file.md (hunk 2 above)
Comment (BornChanger) [reply tikv-configuration-file.md]: > 建议把 quota 下的配置项版本标识都改为 `<span>`，之前是因为前端网站出了问题，暂时不支持 `<span>` 了，所以临时用了（从 v6.0.0 版本开始引入）

Good suggestion

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by tonyxuqqi — APPROVED** [review]

**Comment by TomShawn** [comment]
@BornChanger Can the configs be modified online? If yes, please add them to [the table of this document](https://docs.pingcap.com/zh/tidb/dev/dynamic-config#:~:text=quota.foreground%2Dcpu,%E7%9A%84%E6%9C%80%E5%A4%A7%E6%97%B6%E9%97%B4).

File: dynamic-config.md [hunk 7]
```diff
@@ -166,9 +166,12 @@ show warnings;
...
 | pessimistic-txn.in-memory | 是否开启内存悲观锁功能 |
 | quota.foreground-cpu-time | 限制处理 TiKV 前台读写请求所使用的 CPU 资源使用量，软限制 |
-| quota.foreground-write-bandwidth | 限制事务写入的带宽，软限制 |
-| quota.foreground-read-bandwidth | 限制事务读取数据和 Coprocessor 读取数据的带宽，软限制 |
-| quota.max-delay-duration | 单次前台读写请求被强制等待的最大时间 |
+| quota.foreground-write-bandwidth | 限制前台事务写入的带宽，软限制 |
+| quota.foreground-read-bandwidth | 限制前台事务读取数据和 Coprocessor 读取数据的带宽，软限制 |
+| quota.background-cpu-time | 限制处理 TiKV 后台读写请求所使用的 CPU 资源使用量，软限制 |
+| quota.background-write-bandwidth | 限制后台事务写入的带宽，软限制 |
+| quota.background-read-bandwidth | 限制后台事务读取数据和 Coprocessor 读取数据的带宽，软限制 |
```
Comment (TomShawn) [inline dynamic-config.md:173]:
```suggestion
| quota.background-read-bandwidth | 限制后台事务读取数据和 Coprocessor 读取数据的带宽，软限制 |
| quota.enable-auto-tune | 是否支持 quota 动态调整。如果打开该配置项，TiKV 会根据 TiKV 实例的负载情况动态调整对后台请求的限制 quota |
```

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -1660,35 +1660,71 @@ Raft Engine 相关的配置项。
...
+
+#### `background-read-bandwidth` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
++ 限制后台事务读取数据和 Coprocessor 读取数据的带宽，这是一个软限制。
++ 默认值：0KB（即无限制）
+
+#### `enable-auto-tune` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
++ 是否支持 quota 动态调整。如果打开该配置项，TiKV 会根据 TiKV 实例的负载情况动态调整对后台请求的限制 quota。
++ 默认值：false （即关闭动态调整）
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1727]:
```suggestion
+ 默认值：false（即关闭动态调整）
```

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -1660,35 +1660,71 @@ Raft Engine 相关的配置项。
...
+> **警告：**
+>
+> - 后台限流是 TiDB 在 v6.2.0 中引入的实验特性，不建议在生产环境中使用。
+> - 该功能仅适合在资源有限的环境中使用，以保证 TiKV 在该环境下可以长期稳定地运行。如果在资源丰富的机型环境中开启该功能，可能会导致读写请求量达到峰值时 TiKV 的性能下降的问题。
+
+#### `background-cpu-time` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
++ 限制处理 TiKV 后台读写请求所使用的 CPU 资源使用量，这是一个软限制。
++ 默认值：0（即无限制）
++ 单位：millicpu （当该参数值为 `1500` 时，后端请求会消耗 1.5v CPU）。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1712]:
```suggestion
+ 单位：millicpu（当该参数值为 `1500` 时，后端请求会消耗 1.5v CPU）。
```

File: tikv-configuration-file.md [hunk 10]
```diff
@@ -1660,35 +1660,71 @@ Raft Engine 相关的配置项。
...
+
+#### `background-write-bandwidth` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
```
Comment (TomShawn) [inline tikv-configuration-file.md:1715]:
```suggestion

> **注意：**
>
> 该配置项可以通过 `SHOW CONFIG` 查询到，但暂未生效。设置该配置项的值不生效。
 
```

File: tikv-configuration-file.md [hunk 11]
```diff
@@ -1660,35 +1660,71 @@ Raft Engine 相关的配置项。
...
++ 限制后台事务写入的带宽，这是一个软限制。
++ 默认值：0KB（即无限制）
+
+#### `background-read-bandwidth` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
```
Comment (TomShawn) [inline tikv-configuration-file.md:1720]:
```suggestion

> **注意：**
>
> 该配置项可以通过 `SHOW CONFIG` 查询到，但暂未生效。设置该配置项的值不生效。

```

File: dynamic-config.md [hunk 12]
```diff
@@ -166,9 +166,13 @@ show warnings;
...
```
Comment (TomShawn) [inline dynamic-config.md:173]:
```suggestion
| quota.background-write-bandwidth | 限制后台事务写入的带宽，软限制，暂未生效 |
| quota.background-read-bandwidth | 限制后台事务读取数据和 Coprocessor 读取数据的带宽，软限制，暂未生效 |
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**